tests/test_meshtastic_utils.py

```
@@ -4858,6 +4984,7 @@ def test_connect_meshtastic_ble_interface_creation_timeout(
                 if "BLE interface creation timed out after" in str(call)
             ]
             self.assertEqual(len(error_calls), MAX_TIMEOUT_RETRIES_INFINITE + 1)
+            self.assertTrue(all(call.args[1] == 45.0 for call in error_calls))
```

Ruff (0.15.9)

[warning] Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)